CRM/CiviAwards/Setup/AddManageWorkflowMenu.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use CRM_Civicase_Service_CaseCategoryMenu as CaseCategoryMenu;
+
+/**
+ * Adds the Manage Workflow Menu item for existing Case types.
+ */
+class CRM_CiviAwards_Setup_AddManageWorkflowMenu {
+
+  /**
+   * And Create Manage Workflow Menu for applicant management categories.
+   */
+  public function apply() {
+    CaseCategoryMenu::createManageWorkflowMenuForExistingCaseCategories('applicant_management', TRUE);
```

Comment:
This function should not be called statically once we change the function to a member function.